reconcile/utils/ocm.py

```diff
@@ -931,14 +937,44 @@ def _init_blocked_versions(self, blocked_versions):
                 raise TypeError(f"blocked version is not a valid regex expression: {b}")
 
     @retry(max_attempts=10)
-    def _get_json(self, api):
-        r = requests.get(f"{self.url}{api}", headers=self.headers)
+    def _do_get_request(self, api: str, params: Mapping[str, str]) -> Mapping[str, Any]:
```

Comment:
To avoid confusion, we want to keep inputs general, but for outputs and types on collections, it's encouraged to use concrete types, so:

`_do_get_request(self, api: str, params: Mapping[str, str]) -> dict[str, Any]`

Reply:
done